src/html-imports.js

```diff
@@ -420,11 +391,14 @@
     }
 
     /**
-     * Returns a promise resolved after the loaded imports finish loading scripts
-     * and styles, and fire the load/error events.
-     * @return {!Promise}
+     * Waits for loaded imports to finish loading scripts and styles, then fires
```

Comment:
The name here is a bit confusing. This function "checks if loading is done and then process the imports if so."... maybe something like `processIfDoneLoading`? (not great)